examples/create.rs

```diff
@@ -17,21 +14,23 @@ fn main() {
         process::exit(1);
     }
 
-    let manager = manager::new();
+    let manager = NetworkManager::new();
 
-    let mut devices = device::list(&manager).unwrap();
+    let devices = manager.get_devices().unwrap();
     let device_index = devices
         .iter()
-        .position(|ref d| d.device_type == device::DeviceType::WiFi)
+        .position(|ref d| *d.device_type() == DeviceType::WiFi)
```

Comment:
The `*` is a bit confusing since it is not dereferencing `d`, but the `&DeviceType` returned by `device.device_type()`.